.gitignore

```diff
@@ -1,330 +1,73 @@
-## Ignore Visual Studio temporary files, build results, and
-## files generated by popular Visual Studio add-ons.
-##
-## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
+.vscode/*
```

Comment:
Ensure that macOS/Windows gitignore has been added as well https://github.com/github/gitignore/blob/master/Global/macOS.gitignore
https://github.com/github/gitignore/blob/master/Global/Windows.gitignore

Reply:
Fixed in 7e193fa0eb80932f32f1b6a02fc292ef5adb855d